wd_blender_tools/addon_static.py

```diff
@@ -5,7 +5,7 @@
 
 """Static data for the add-on except for text messages. Those are in lib/text_static"""
 
-blender_addon_version = (1, 0, 0)
+blender_addon_version = (1, 0, 1)
```

Comment:
Hey @MilutinSekulic  is this metadata or addon version?

Reply:
That one is metadata. Although we were, at the time, treating both the same for simplicity. As agreed we will be separating it.